Balkanized Vrykul

zumbak04 opened the pull request:
[**_HOW TO REVIEW LOCALIZATION?_**](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/How-to-be-Keeper-(Tester)#reviewing-code)
[**_CLICK HERE TO SEE LOCALIZATION_**](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1039/files?file-filters%5B%5D=.csv#diff-daa3272550baca2c60c573c20a98f7d9)

<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Vrykul of the Howling Fjord have been balkanized.
- Angerboda, Ymiron's ex-wife, can use **Awaken King Ymiron** decision to awaken Ymiron who gets the Howling Fjord title/claim, army and commanders.
- Added flag for the Winterskorn dynasty and title.
- Added flag for the Dragonflayer dynasty.
- Skeld Drakeson, chief of Ember Clutch, is tribal now.

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Purged `d_dragonflayer` title since it's titular and AI doesn't know what to do with it.

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
- Just try to play Angerboda and any other vrykul of the Howling Fjord, tell what you think.
- @spacsea and @DukeJeans could you review added localization. I left 2 links for you above. The first link leads to manual (if you need it), the second leads to localization itself.

geffe1 commented:
>Skeld Drakeson, chief of Ember Clutch, is feudal now.

Wrong, tribal monarchy. But so is all other vrykul, so I see no issue with you not writing it correct.

>Added flag for the Dragonflayer dynasty.

Yepp.

>Added flag for the Winterskorn dynasty and title.

I suppose you mean this title?
<details>
<summary>Click to expand</summary>
 
![ck2_492](https://user-images.githubusercontent.com/31706375/71644431-8c785880-2cc8-11ea-94e1-8c8fa7d06f18.png)
</details>


>Angerboda, Ymiron's ex-wife, can use Awaken King Ymiron decision to awaken Ymiron who gets claims, army and commanders.

He doesn't get claims, he gets the Kingdom dejure title though, which I think is fine.

<details>
<summary>Click to expand</summary>

![ck2_494](https://user-images.githubusercontent.com/31706375/71644460-f0028600-2cc8-11ea-9569-14dc6f6b3b69.png)
</details>

>Vrykul of the Howling Fjord have been balkanized.

Yepp

Found one issue though, the Vrykul should clearly not think like that. Either "oh shit" or "hurray".
<details>
<summary>Click to expand</summary>

 ![ck2_488](https://user-images.githubusercontent.com/31706375/71644477-466fc480-2cc9-11ea-9d02-96c45cccaaab.png)
</details>

geffe1 commented:
Add an event to influence vrykul in dejure northrend through dreams to join the Scourge and thier "death god", make certain traits- like cruel, cynical among others increase the chance for the AI to join, while stubborn and kind decreases the chance. Could fire after the Lich King's arrival in Northrend.

zumbak04 commented:
> > Skeld Drakeson, chief of Ember Clutch, is feudal now.
> 
> Wrong, tribal monarchy. But so is all other vrykul, so I see no issue with you not writing it correct.

Poor wording, fixed it.

> > Added flag for the Winterskorn dynasty and title.
> 
> I suppose you mean this title?

I see the Winterskorn dynasty and title on your screenshot, though you opened the character interface of Winterskorn vassal... so I don't know what title you mean...

> He doesn't get claims, he gets the Kingdom dejure title though, which I think is fine.

Poor wording, fixed it.

> Found one issue though, the Vrykul should clearly not think like that. Either "oh shit" or "hurray".

What about either **Hail the King** or **Who cares**? Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1039/commits/41b9bd76003f030f3894c8d13887f37f7636d825.

geffe1 commented:
> > > Skeld Drakeson, chief of Ember Clutch, is feudal now.
> > 
> > 
> > Wrong, tribal monarchy. But so is all other vrykul, so I see no issue with you not writing it correct.
> 
> Poor wording, fixed it.
> 
> > > Added flag for the Winterskorn dynasty and title.
> > 
> > 
> > I suppose you mean this title?
> 
> I see the Winterskorn dynasty and title on your screenshot, though you opened the character interface of Winterskorn vassal... so I don't know what title you mean...
> 
> > He doesn't get claims, he gets the Kingdom dejure title though, which I think is fine.
> 
> Poor wording, fixed it.
> 
> > Found one issue though, the Vrykul should clearly not think like that. Either "oh shit" or "hurray".
> 
> What about either **Hail the King** or **Who cares**? Done in [41b9bd7](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/41b9bd76003f030f3894c8d13887f37f7636d825).

I'd say go with "Hail to the King", though wouldn't it make more sense then for these vrykul choosing that, to become a part of his new kingdom?
https://www.youtube.com/watch?v=DelhLppPSxY

zumbak04 commented:
> I'd say go with "Hail to the King", though wouldn't it make more sense then for these vrykul choosing that, to become a part of his new kingdom?

Nah, will become too easy. Actually, **Hail the King** isn't real thing, just my expression. In-game it sounds like **The King has returned!** if vrykul is from the Howling Fjord.

geffe1 commented:
> > I'd say go with "Hail to the King", though wouldn't it make more sense then for these vrykul choosing that, to become a part of his new kingdom?
> 
> Nah, will become too easy. Actually, **Hail the King** isn't real thing, just my expression. In-game it sounds like **The King has returned!** if vrykul is from the Howling Fjord.

Tsk, too easy? But the Scourge gets a ton of free armies randomly and stuff. That's also too easy.

I think it would be cool if the dejure howling fjord ones gets an option to "Hail to the King, kneel to the crown", though that is up to you.

The rest of the vrykul? Should probably think "Who cares?".

zumbak04 commented:
> Tsk, too easy? But the Scourge gets a ton of free armies randomly and stuff. That's also too easy.
> 
> I think it would be cool if the dejure howling fjord ones gets an option to "Hail to the King, kneel to the crown", though that is up to you.

Ok, what's more interesting: vrykul joining you via event or you uniting vrykul via war?

geffe1 commented:
>Ok, what's more interesting: vrykul joining you via event or you uniting vrykul via war?

Well, that depends. As he gets free troops, thus making it easy to win most wars either way.

zumbak04 commented:
> Well, that depends. As he gets free troops, thus making it easy to win most wars either way.

I can remove it if you want.
![image](https://user-images.githubusercontent.com/46384992/71685361-81e5be00-2db1-11ea-93dd-a0293faf4f4c.png)
These troops don't reinforce, so you can't use them to conquer the whole Howling Fjord.

geffe1 commented:
> > Well, that depends. As he gets free troops, thus making it easy to win most wars either way.
> 
> I can remove it if you want.
> ![image](https://user-images.githubusercontent.com/46384992/71685361-81e5be00-2db1-11ea-93dd-a0293faf4f4c.png)
> These troops don't reinforce, so you can't use them to conquer the whole Howling Fjord.

is ok :( I'm sad now, calling in sick because you don't like my ideas.

DukeJeans commented on localisation/00_zuwc_united_localisation.csv:
```diff
@@ -3013,6 +3013,13 @@ RULE_STEREOTYPICAL_TRAITS_LESS_25;Less Frequent;;;;;;;;;;;;;x
+EVTOPTA_WCRLM_800;The King has returned!;;;;;;;;;;;;;x
+EVTOPTB_WCRLM_800;Ymiron? Many legends of him have been told...;;;;;;;;;;;;;x
+awaken_ymiron;Awaken King Ymiron;;;;;;;;;;;;;x
+awaken_ymiron_desc;Awaken King Ymiron, a legendary leader of the Dragonflayers, he ruled the lands now called "the Howling Fjord" a long time ago. If you click this decision, you will start playing him.;;;;;;;;;;;;;x
```
```suggestion
awaken_ymiron_desc;Awaken King Ymiron, the legendary leader of the Dragonflayers who ruled the land known as the Howling Fjord a long time ago. If you click this decision, you will start playing him.;;;;;;;;;;;;;x
```

MemeWizard commented on events/wc_realm_events.txt:
```diff
@@ -545,4 +545,158 @@ character_event = {
+			}
+			
+			# Ymiron gets ROOT titles, k_howling_fjord or claim on k_howling_fjord
+			event_target:target_ymiron = { ROOT = { this_abdicate_transfer_wealth_to_play_prev_effect = yes } }
```
Don't transfer wealth, don't transfer artifacts.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768428-eaa27700-2f26-11ea-8b27-1af8122a92b3.png)
![image](https://user-images.githubusercontent.com/46576860/71768437-0279fb00-2f27-11ea-80a0-67505cb5f134.png)
![image](https://user-images.githubusercontent.com/46576860/71768451-09087280-2f27-11ea-8045-e48adcb8e9d1.png)

</details>

MemeWizard replied:
Also don't clear claims

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768928-1aa04900-2f2c-11ea-84d6-17167a76d183.png)

</details>

zumbak04 replied:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/f8fe146924d610f47a4280d438eb7ead907d5708 and https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/fb8761cd07dfb839fbf2ac1c71a4667be23e6134.

MemeWizard commented on events/wc_realm_events.txt:
```diff
@@ -545,4 +545,158 @@ character_event = {
+				type = ymiron
+			}
+			
+			# Angerboda becomes your spouse
```
`event 500` fires for Angerboda

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768479-45d46980-2f27-11ea-81f3-af521bd6506b.png)
![image](https://user-images.githubusercontent.com/46576860/71768480-4a008700-2f27-11ea-9120-ae3fc00cdb10.png)


</details>

zumbak04 replied:
And what?

MemeWizard replied:
And maybe you should think at least once in a while.
This event shouldn't fire for landless character, shouldn't give gold/prestige to Angerboda instead of Ymiron, probably shouldn't fire at all for either of them, since it's not new marriage.

zumbak04 replied:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/33c1a3e573edf90c9e5e78ac3def329593a61f43.

MemeWizard commented on events/wc_realm_events.txt:
```diff
@@ -545,4 +545,158 @@ character_event = {
+			}
+			
+			# Angerboda becomes your spouse
+			c_100001 = {
```
Will marry even if this character can't marry

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768853-728a8000-2f2b-11ea-84ab-06738f7fcbeb.png)
![image](https://user-images.githubusercontent.com/46576860/71768862-7ae2bb00-2f2b-11ea-8b89-f3505cf5b800.png)
![image](https://user-images.githubusercontent.com/46576860/71768866-81713280-2f2b-11ea-8f30-c4c8e95f714d.png)

</details>

zumbak04 replied:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/280aecdd7daba1dec5ad0f59bddc472cf2d9bc47.
